golismero/common.py

```diff
@@ -1149,6 +1152,36 @@ def cookie(self, cookie):
     #--------------------------------------------------------------------------
 
     @property
+    def data(self):
+        return self._data
+
+    @data.setter
+    def data(self, data):
+        if data:
+            # Parse the data argument.
+            try:
+                if isinstance(data, unicode):
+                    data = data.encode("UTF-8")
+
+                if "=" in data:
+                    # Prepare data.
+                    data = data.replace("=", ":")
```

Comment:
There are Python-standard ways to parse POST parameters, I would prefer to use those instead of rolling out our own parsing code each time we need to do this. :)